### PR TITLE
Add cross-platform capability for Windows OS support

### PR DESCRIPTION
**Pull request by orangedeng**
refer to https://github.com/rancher/rancher/issues/10449

### CODE REVIEW

**Comment by orangedeng** [comment]
@yasker

File: scripts/package [hunk 1]
```diff
@@ -1,23 +1,35 @@
-#!/bin/bash
+#!/bin/bash -x
```
Comment (cjellick) [inline scripts/package]: Dont add debug mode to script

File: scripts/package [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
-cd $(dirname $0)/../package
+  [ "$os"   != "linux" ] && suffix="${suffix}_${os}"
+  [ "$arch" != "amd64" ] && suffix="${suffix}_${arch}"
 
-TAG=${TAG:-${VERSION}${SUFFIX}}
-REPO=${REPO:-rancher}
+  if $(echo ${tag} | grep -q dirty); then
+      tag=dev
+  fi
+  tag="${VERSION}${suffix}"
```
Comment (cjellick) [inline scripts/package]: What kind of tag is this going to produce? Examples please.

Comment (cjellick) [reply scripts/package]: include the image names too

**Comment by alena1108** [comment]
@orangedeng Can you re-push to trigger Drone build?

File: scripts/package (hunk 2 above)
Comment (orangedeng) [reply scripts/package]: As @deniseschannon comments in https://github.com/rancher/rancher-catalog/pull/1026, I will use the same tag as linux version. I think that the image name will be like `rancher/metadata-windows:<tag>` if Denise agrees.

**Comment by orangedeng** [comment]
Updated @cjellick @deniseschannon

**Comment by cjellick** [comment]
LGTM

**Pull request merged**